### PR TITLE
feat(AR-61): 💄 adds redirection to spaces list logic on header logo click

### PR DESCRIPTION
**Pull request by abenmassaoud**
## Pull request type

Jira Link: [AR-61](https://storyblok.atlassian.net/browse/AR-61)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

https://user-images.githubusercontent.com/16685155/216122691-2e128fc0-da3f-4a23-9488-6574fa3588ea.mov

## What is the new behavior?

- The `<SbSidebar/>` component now receives a new **optional** `logoDestinationUrl` prop.

## Other information

This PR is linked to [another PR on the storyfront repo.](https://github.com/storyblok/storyfront/pull/3355)

[AR-61]: https://storyblok.atlassian.net/browse/AR-61?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by fernandacanto — CHANGES_REQUESTED** [review]
Hi @abenmassaoud, the code is fine, but it's interesting to put an example of use in Sidebar.stories.js. ;)

File: src/components/Sidebar/Sidebar.vue [hunk 1]
```diff
@@ -14,13 +14,28 @@
...
+        >
+          <img
+            v-if="logo"
+            class="sb-custom-logo"
+            :src="logo"
+            alt="Custom Sidebar Logo"
+          />
+          <SbSidebarLogo v-else variant="dark" />
+        </SbLink>
+        <template v-else>
```
Comment (fariacc) [inline src/components/Sidebar/Sidebar.vue:30]: I don't think it's necessary to wrap these elements inside a template tag if you use `v-if-else` for the image

**Review by fariacc — APPROVED** [review]

File: src/components/Sidebar/Sidebar.vue (hunk 1 above)
Comment (fariacc) [reply src/components/Sidebar/Sidebar.vue:30]: Oh I see, thanks for the explanation my friend 🚀

**Comment by abenmassaoud** [comment]
> Hi @abenmassaoud, the code is fine, but it's interesting to put an example of use in Sidebar.stories.js. ;)

Hi @fernandacanto,
Thank you for the feedback. Indeed an example helps.
I added it [here](https://storyblok-design-system-k8dk8ixk8-storyblok-com.vercel.app/?path=/story/design-system-components-sbsidebar--logo-destination-url) 🙂

**Pull request merged**